files/usr/share/jeos-firstboot/jeos-firstboot-functions

```diff
@@ -110,11 +124,18 @@ findlocales()
         list=()
         local l locale
         # List only locales which are both in live-langset-data and glibc-locale(-base)
-        for l in /usr/share/langset/*; do
-                locale="${l#/usr/share/langset/}"
-                [ -d "/usr/lib/locale/${locale}.utf8" ] || continue
-                list+=("${locale}" '')
-        done
+        if [ -d /usr/share/langset ]; then
```

Comment:
jeos-firstboot calls langset.sh so this is a hard requirement

Reply:
yeah, was good enough for dry run though